moped-toolbox/populate_geometry_transform/transpose_geometry.py

```diff
@@ -54,6 +57,26 @@ def execute(sql, values, get_result=False):
         return True
 
 
+def remove_deprecated_fields(fields, values):
+    deprecated_fields = [
+        "source_layer",
+        "project_extent_id",
+        # "symbol",
+        # "line_type",
+        # "ctn_segment_id",
+        # "to_address_max",
+        # "from_address_min",
+        # "full_street_name"
```

Comment:
Are these fields that are best left in? No issue with the commented field names, I'm just curious.

Reply:
agreed!